src/controllers/QuestionnaireController.ts

```diff
@@ -51,4 +53,120 @@ export class QuestionnaireController {
             }
         );
     }
+
+    /**
+     * Add a questionnaire.
+     *
+     * @param {Request} req
+     * @param {Response} res
+     * @memberof QuestionnaireController
+     */
+    @Post('')
+    @Middleware(
+        jwt({
+            secret: AuthConfig.jwtSecret,
+            algorithms: ['HS256'],
+            requestProperty: 'payload',
+            isRevoked: AuthorizationController.checkApiUserLogin
+        })
+    )
+    public async addQuestionnaire(req: Request, res: Response) {
+        const url = req.body.url;
```

Comment:
Since you use POST and not GET, you can't get url, version, etc. this way. Use query instead of body

Reply:
No, it is the other way round: [req.query](https://expressjs.com/de/api.html#req.query) and[ req.body](https://expressjs.com/de/api.html#req.body) ?

Reply:
The openapi definition was wrong, so when using swaggerui, the data would be sent with the reqeuest parameters and not in the body